src/ipc/ipc4/handler.c

```diff
@@ -536,16 +536,20 @@ static int ipc_wait_for_compound_msg(void)
 {
 	int try_count = 30;
 
+	pm_runtime_disable(CORE_HP_CLK, PLATFORM_PRIMARY_CORE_ID);
```

Comment:
Is this still needed? Will this work correctly with KCPS which changes clock during IPC?